org.freedesktop.Sdk.Extension.vala.appdata.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2022 Lorenz Wildberg <lorenz@wild-fisch.de> -->
+<component type="addon">
```

Comment:
`addon`-type components should have `<extends></extends>` tag, indicating what they're extension for (`org.freedesktop.Sdk` in this case).

Also, this file has tab/spaces mixture.

Reply:
Thanks!